readme.md

```diff
@@ -79,7 +79,7 @@ Next run the appropriate build shell script for your platform:
 Finally, once you have finished the build steps (on Windows, Mac or Linux) you
 should add the ``bin`` directory to your PATH.
 
-See also [rebuilding the compiler](intern.html#rebuilding-the-compiler).
+See also [rebuilding the compiler](doc/intern.rst#rebuilding-the-compiler).
```

Comment:
should be: https://nim-lang.github.io/Nim/intern.html#rebuilding-the-compiler

Reply:
For the scope of the repo, that link is correct and is consistent with all other usage. We don't publish this readme file as a webpage anywhere.